server/test/api/debtors.js

```diff
@@ -45,4 +54,26 @@ describe('(/debtors) The /debtors API', function () {
       })
       .catch(helpers.handler);
   });
+
+  it('GET /debtors should return a list of all debtors', function () {
+    return agent.get('/debtors')
+      .then(function (res) {
+        /** @fixme need use helpers.api.listed but data are provided from other test modules */
+        expect(res.body).to.not.be.empty;
```

Comment:
In the future, `expect(res.body).to.have.length.at.least(1)` is a stronger test that `expect(res.body).to.not.be.empty`.  You do not need `expect(res.body).to.not.be.empty` because of the second test.